### PR TITLE
feat: Contacts remove

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/ticket/issues/979

Need to remove contacts from the application

## How to test:

Work through the flow of the application.

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: src/services/ContactRepository.ts [hunk 1]
```diff
@@ -13,30 +13,6 @@ import { notifyFailure } from './FeedbackService'
...
-          error,
-          message: `Could not resolve contacts'`,
-          origin: 'ContactRepository.findAll()',
-          type: 'ERROR.FETCH.GET',
-        })
-        return error
-      })
-  }
-
   public static async findByAddress(
```
Comment (tjwelde) [inline src/services/ContactRepository.ts:16]: Do we need this function (and the others, too)?

**Review by tjwelde — CHANGES_REQUESTED** [review]
looks pretty good so far. we might also need to remove stuff from the services, though

File: src/services/ContactRepository.ts (hunk 1 above)
Comment (Dudleyneedham) [reply src/services/ContactRepository.ts:16]: We used it to fetch all the contacts from the store. I removed all the places we use it. `Add`, `getContactFromIdentity`, `ImportViaDid and `findByAddress` I feel these will be used with the `DID` store.

Comment (tjwelde) [reply src/services/ContactRepository.ts:16]: yeah, I am not talking about the function you actually removed, but the `findByAddress` and others. please follow the traces and see, if they or code inside of them can be removed

**Comment by tjwelde** [comment]
I made new tickets for the services stuff

**Comment by tjwelde** [comment]
Screenshot 2021-01-27 at 17 00 40

Can't request terms from attester

**Review by tjwelde — APPROVED** [review]
LGTM

**Pull request merged**